### PR TITLE
Added functional tests for uncovered DeltaSpike quickstarts

### PR DESCRIPTION
**Pull request by rsmeral**
Depends on updated Graphene and Drone version in wfk bom https://github.com/jboss-developer/jboss-wfk-boms/pull/3

### CODE REVIEW

**Comment by sgilda** [comment]
I don't see any updated instructions in the README files for the quickstarts impacted by this update. Could you add a section with instructions on how to run the functional tests, similar to the ones added in the following commit? 
https://github.com/maschmid/jboss-wfk-quickstarts/commit/4f22b627466c6716d07a4073de11f5e2f3b7b01b

**Comment by rsmeral** [comment]
Thanks for the reminder, I totally forgot.

**Comment by sgilda** [comment]
Thanks @rsmeral . :-)

**Comment by pmuir** [comment]
Code review passes, this is great stuff :-)

I tested the various tests, and I get errors on deltaspike authorization:

```
Failed tests:   loginTest(org.jboss.wfk.test.deltaspike.authorization.AuthorizationTest)

Tests in error: 
  employeeTest(org.jboss.wfk.test.deltaspike.authorization.AuthorizationTest): Cannot locate element using: input[value='Employees only Method']
  employeeTest(org.jboss.wfk.test.deltaspike.authorization.AuthorizationTest): Cannot locate element using: input[value='Logout']
  loginTest(org.jboss.wfk.test.deltaspike.authorization.AuthorizationTest): Cannot locate element using: input[value='Logout']
  adminTest(org.jboss.wfk.test.deltaspike.authorization.AuthorizationTest): Cannot locate element using: input[value='Admins only Method']
  adminTest(org.jboss.wfk.test.deltaspike.authorization.AuthorizationTest): Cannot locate element using: input[value='Logout']
```

Once this is fixed, I can merge this.

**Comment by sgilda** [comment]
When I test deltaspike-authorization, I get a different result. 

[INFO] No tests to run.
[INFO] Surefire report directory: /home/sgilda/GitRepos/jboss-wfk-quickstarts/deltaspike-authorization/target/surefire-reports

---

##  T E S T S

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

Looking into it further.

File: deltaspike-authorization/README.md [hunk 1]
```diff
@@ -83,6 +83,26 @@ Undeploy the Archive
...
+2. Build the quickstart WAR using the following command:
+
+        mvn clean package
+
+3. Navigate to the functional-tests/ directory in this quickstart.
+4. If you have a running instance of the JBoss Server, as described above, run the remote tests by typing the following command:
+
+        mvn clean verify -Parq-jbossas-remote
+
+5. If you prefer to run the functional tests using managed instance of the JBoss server, meaning the tests will start the server for you, type fhe following command:
```
Comment (sgilda) [inline deltaspike-authorization/README.md]: When they run the tests using a managed instance of the server, don't they need to configure the `jbossHome` property in the arquillian.xml file to point to the server path? The instructions are in the root README file, but you might want to point them there with something like:

_NOTE: The following commands assume you have configured your Maven user settings. If you have not, you must include Maven setting arguments on the command line. See [Run the Arquillian Tests](../README.md#run-the-arquillian-tests) for complete instructions and additional options._

File: deltaspike-beanmanagerprovider/README.md [hunk 2]
```diff
@@ -82,6 +82,27 @@ Undeploy the Archive
...
+2. Build the quickstart WAR using the following command:
+
+        mvn clean package
+
+3. Navigate to the functional-tests/ directory in this quickstart.
+4. If you have a running instance of the JBoss Server, as described above, run the remote tests by typing the following command:
+
+        mvn clean verify -Parq-jbossas-remote
+
+5. If you prefer to run the functional tests using managed instance of the JBoss server, meaning the tests will start the server for you, type fhe following command:
```
Comment (sgilda) [inline deltaspike-beanmanagerprovider/README.md]: Same issue as above and for all the readme files.

**Comment by rsmeral** [comment]
@pmuir Fixed, thanks. Wrong password was used.

@sgilda That happens when you try to execute the test goal on the quickstart project. The test needs to be run in the functional-tests project. It is separated this way to avoid running the functional test by accident and to emphasize that the functional tests are something that needs special conditions (a server, a browser and an already compiled artifact to test).

**Comment by rsmeral** [comment]
@sgilda re:README: I updated the README as you suggested. Also, I checked the "Run the Arquillian tests" section which the note points to and updated that as well (https://github.com/jboss-developer/jboss-wfk-quickstarts/pull/16). WDYT?

**Comment by rsmeral** [comment]
Also, if the README fix here is accepted, please fix also https://github.com/jboss-developer/jboss-wfk-quickstarts/blob/master/kitchensink-deltaspike/README.md. Thanks.

**Comment by pmuir** [comment]
Merged.

**Pull request closed without merging**